src/client/src/talker_client.cpp

```diff
@@ -0,0 +1,42 @@
+#include <string>
+#include <signal.h>
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Found C system header after C++ system header. Should be: talker_client.h, c system, c++ system, other.  [build/include_order] [4]